### ISSUE TITLE
libphutil hash is outdated

### ISSUE THREAD

**Issue by justinj** [Homebrew/homebrew-php#2480]
```
± % brew reinstall arcanist --HEAD                                                !10531
==> Reinstalling homebrew/php/arcanist
==> Cloning https://github.com/phacility/arcanist.git
Updating /Library/Caches/Homebrew/arcanist--git
==> Checking out branch master
==> Downloading https://github.com/phacility/libphutil/archive/master.tar.gz
Already downloaded: /Library/Caches/Homebrew/arcanist--libphutil-HEAD.tar.gz
Error: SHA256 mismatch
Expected: 71fe72bd0be33f365ee07979ae335261d471fe523e58f7563a951db9deee54f2
Actual: 9e85b459129ee33e7e73e1e4ae5766b865398319817c723acbf49850ad5a2ede
Archive: /Library/Caches/Homebrew/arcanist--libphutil-HEAD.tar.gz
To retry an incomplete download, remove the file above.
```

Same thing happens after deleting the file.

I see there was some conversation about this [here](https://github.com/Homebrew/homebrew-php/commit/096e6751bee789b5b202a9cf83363f4e4a622167), which is probably relevant.

**Comment by josegonzalez** [comment]
We could update it to use a specific sha: https://github.com/phacility/libphutil/archive/d5b4421792e5829b7a760c55dbf9adbc70669455.zip

**Comment by justinj** [comment]
Seems fixed by #2482, thanks!
